pages/common/c99.md

```diff
@@ -0,0 +1,19 @@
+# C99
+
+> Compiles C programs according to the ISO C standard.
+
+- Compile source files and create an executable:
```

Comment:
"file(s)" sounds good to me, it makes it clear without making the example longer than necessary.